docs/getting-started/overview.md

```diff
@@ -77,7 +77,9 @@ mysql -h 127.0.0.1 -P 4002
 
 ## Create table
 
-Let's start by creating the `system_metrics` table which contains system resource metrics, including CPU/memory/disk usage. The data is scraped every 5 seconds.
+**Note: GreptimeDB provides schemaless writing that you don't need to create table manually with other protocols. See [write data](/user-guide/write-data.md).**
```

Comment:
Can you make a section in `write data` that describe auto schema? So we can link it here, using, for example, `/user-guide/write-data.md#auto_schema`